namerd/iface/mesh/src/main/scala/io/buoyant/namerd/iface/MeshIfaceInitializer.scala

```diff
@@ -0,0 +1,46 @@
+package io.buoyant.namerd
+package iface
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.{Path, Namer, Service, Stack}
+import com.twitter.finagle.buoyant.{H2, h2}
+import com.twitter.finagle.naming.NameInterpreter
+import com.twitter.finagle.stats.StatsReceiver
+import com.twitter.util.Duration
+import com.twitter.util.TimeConversions._
+import io.buoyant.grpc.runtime.ServerDispatcher
+import java.net.InetSocketAddress
+
+class MeshIfaceConfig extends InterfaceConfig {
```

Comment:
I couldn't figure out a way to configure the mesh interface to serve over TLS. Maybe it's not supported yet? Or am I missing the server TLS option? Interpreter client TLS seems to work.